tests/one_taskset.rs

```diff
@@ -27,14 +27,16 @@ fn test_single_taskset() {
         .return_body("<HTML><BODY>about page</BODY></HTML>")
         .create();
 
-    crate::GooseAttack::initialize_with_config(common::build_configuration())
+    let _stats = crate::GooseAttack::initialize_with_config(common::build_configuration())
```

Comment:
Technically it's a goose attack and not stats ...

Reply:
fixed